lib/plugins/aws/package/compile/events/httpApi/index.js

```diff
@@ -48,9 +48,27 @@ class HttpApiEvents {
       },
     };
 
-    // TODO: Complete schema, see https://github.com/serverless/serverless/issues/8021
     this.serverless.configSchemaHandler.defineFunctionEvent('aws', 'httpApi', {
-      anyOf: [{ type: 'string' }, { type: 'object' }],
+      anyOf: [
+        { type: 'string' },
+        {
+          type: 'object',
+          properties: {
+            method: { type: 'string' },
+            path: { type: 'string' },
+            authorizer: {
+              type: 'object',
+              properties: {
+                id: { type: 'string' },
+                name: { type: 'string' },
+                authorizationScopes: { type: 'array', items: { type: 'string' } },
+              },
+              oneOf: [{ required: ['id'] }, { required: ['name'] }],
+            },
+          },
+          required: ['method', 'path'],
```

Comment:
`method` is not required when `path: *`, therefore I believe we should leave just `path` here